src/main/java/stream/flarebot/flarebot/Events.java

```diff
@@ -61,12 +64,19 @@ public Events(FlareBot bot) {
     @Override
     public void onReady(ReadyEvent event) {
         FlareBot.getInstance().latch.countDown();
+        new Timer().schedule(new TimerTask() { //If you're not ok with this in the ready event just say so.
```

Comment:
* Don't use timer, we have our own scheduler
* Don't just throw the map away, it's going to take more RAM
* Explain why do you not use the expiring map